docs/yul.rst

```diff
@@ -931,6 +931,8 @@ the ``dup`` and ``swap`` instructions as well as ``jump`` instructions, labels a
 +-------------------------+-----+---+-----------------------------------------------------------------+
 | difficulty()            |     | F | difficulty of the current block (see note below)                |
 +-------------------------+-----+---+-----------------------------------------------------------------+
+| prevrandao()            |     | P | randomness provided by the beacon chain (see note below)        |
```

Comment:
Does the "note below" not need any updating? Might be worth a quick mention there that ``difficulty()`` will be disallowed in >=paris, etc.

Reply:
Indeed. Updated it.